src/main/resources/com/puppycrawl/tools/checkstyle/messages_zh.properties

```diff
@@ -8,6 +8,13 @@ DefaultLogger.addException=检查错误： {0}
 DefaultLogger.auditStarted=开始检查……
 DefaultLogger.auditFinished=检查完成。
 
+#
+# for Main.java
+#
+Main.errorCounter=Checkstyle以 {0} 個錯誤結束。
+Main.loadProperties=無法從文件 ''{0}'' 加載屬性。
+Main.createListener=輸出格式無效 找到 ''{0}''，但預期為 ''{1}'' 或 ''{2}'' 。
```

Comment:
Almost good, expect that [Simplified Chinese](https://en.wikipedia.org/wiki/Simplified_Chinese_characters) is more common. So my suggestion:
```
Main.errorCounter=Checkstyle以 {0} 个错误结束。
Main.loadProperties=无法从文件 ''{0}'' 中加载属性。
Main.createListener=非法的输出格式。预期为 ''{1}'' 或 ''{2}''，但实际为 ''{0}''。
```